src/node/src/command.rs

```diff
@@ -126,6 +127,18 @@ pub enum DB3Command {
         public_grpc_url: String,
     },
 
+    /// Start db3 indexer
+    #[clap(name = "indexer")]
+    Indexer {
+        /// the url of db3 grpc api
+        #[clap(long = "url", global = true, default_value = "http://127.0.0.1:26659")]
+        public_grpc_url: String,
+        #[clap(short, long, default_value = "./indexer.db")]
```

Comment:
db_path is a folder but indexer.db is like a file.

Reply:
just wanna be consistent with https://github.com/dbpunk-labs/db3/blob/feat%2Fadd_indexer/src/node/src/command.rs#L169
faucet.db.

I thought it was the pattern for db path

Reply:
the faucet uses a `single file` storage engine redb so I used a '.db' subfix